_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,438 @@
+  allowing the unused code to be removed from later node software
+  releases.
+
+  This activation method was first tested with the soft fork that added
+  [BIP68][] consensus-enforced sequence numbers, [BIP112][]
+  `OP_CHECKSEQUENCEVERIFY`, and [BIP113][] nLockTime enforcement by
+  median time past.  The fork quickly made it to the locked-in phase,
+  and then proceeded automatically to activation.
+
+  #### [2016-7] BIP9, BIP148, and BIP91: the BIP141/143 segwit activation
+
+  The segwit soft fork was implemented to use the [BIP9][] activation
+  method.  A few miners quickly began signaling readiness but support
+  peaked far below the needed 95% threshold.  Some Bitcoin users felt
+  miners were illegitimately delaying a useful new feature and supported
+  a mandatory activation mechanism specified in [BIP148][].  The final
+  form of BIP148 required any node following its rules to reject any
+  blocks that didn't signal support for BIPs 141/143 starting on a
+  certain date.
+
+  BIP148 created the risk of a consensus failure.  If miners failed to
+  follow the BIP148 rules and most users continued to consider those
+  miners' blocks as valid, users of BIP148 might accept bitcoins that
+  most other Bitcoin users would consider invalid.  Alternatively, if
+  most users of Bitcoin followed the BIP148 rules---but miners still
+  produced many BIP148-invalid blocks---those users who didn't enforce
+  the BIP148 could accept bitcoins that the BIP148 users would consider
+  invalid.  Only if miners were persuaded to follow the BIP148 rules (or
+  if no users tried to enforce the BIP148 rules) would safety be
+  assured.
+
+  Around the time BIP148 began receiving significant public support,
+  everal miners, exchanges, and other businesses signed their support
+  for a two-step proposal that started with activation of segwit in a
+  way that would follow the BIP148 rules.  This first stage was proposed
+  in [BIP91][], which was a modification of the BIP9 rules.  Miners used
+  a BIP9 versionbit to signal whether they would enforce a temporary
+  rule to reject any blocks not signaling the BIP141/143 segwit
+  versionbit.  Compared to BIP9, the BIP91 threshold was reduced from
+  95% to 80% and its monitoring and lockin period lengths were reduced
+  from 2,016 blocks to 336 blocks.
+
+  BIP91 locked in and activated almost as soon as was possible.
+  Subsequently, BIP141/143 locked in and activated.  BIP148 mandatory
+  signaling expired when BIP141/143 activated.  The second stage of
+  the proposal from miners, exchanges, and other businesses required a
+  hard fork; spokespersons for the signers eventually withdrew their
+  proposal after vocal opposition from a large number of individual
+  users and several businesses.
+
+  It remains a subject of debate exactly how much each of the above
+  events, and other events occuring around the same time, actually
+  contributed to segwit being activated.
+
```

Comment:
BIP 9 was successful in the presence of BIP 148 and BIP 91. I don't know if it can be called "successful" if it needed BIP 148 and BIP 91 to get it safely over the line. Ideally we wouldn't need the use of three BIPs to get a soft fork activated :)